packages/data-context/package.json

```diff
@@ -14,10 +14,21 @@
   },
   "dependencies": {
     "@storybook/csf-tools": "^6.4.0-alpha.38",
+    "@urql/core": "2.3.1",
+    "@urql/exchange-execute": "1.1.0",
+    "@urql/exchange-graphcache": "4.3.3",
+    "chokidar": "3.5.1",
     "create-cypress-tests": "0.0.0-development",
     "cross-fetch": "^3.1.4",
     "dataloader": "^2.0.0",
+    "dedent": "^0.7.0",
+    "electron": "14.1.0",
```

Comment:
Must've missed when this happened. I don't think we should be depending on `electron`, if anything it should be injected from server.

Reply:
Yeah I agree, I thought it was a bit weird a data layer pulls in electron - we should remove that. I think @ImCesar is working around electron now, maybe he can remove the reference to electron we have in `data-context` when he is back.